unlock-app/src/config.js

```diff
@@ -71,7 +71,7 @@ export default function configure(
   let requiredNetwork = 'Dev'
   let requiredNetworkId = 1984
   let requiredConfirmations = 12
-  let unlockAddress = ''
+  let unlockAddress = '0x8DE3f95E2efd3B9704ccb0d0925EC951bC78cb8B' // Unlock address by default (smart contract deployments yield the same address on a "clean" node as long as long as the migration script runs in the same order)
```

Comment:
Put the comment on the previous line for line breaks?

Reply:
sure (note that this is a draft PR, just to get feedback ;P)